### ISSUE TITLE
SASS compilation error on master

### ISSUE THREAD

**Issue by jmcdonagh** [instructure/canvas-lms#851]
### Summary:
I'm building a canvas container and have run into an issue with SASS compilation while running 'bundle exec rake canvas:compile_assets':

```
81a829f4671a:~/canvas$ RAILS_ENV=testing bundle exec rake canvas:compile_assets
--> Starting: 'Making sure node_modules are up to date'
--> Finished: 'Making sure node_modules are up to date' in 9.443557515
--> Starting: 'compile css (including custom brands)'
compiled bundles/all_courses.scss legacy_normal_contrast  in 376 ms
compiled bundles/all_courses.scss legacy_high_contrast  in 381 ms
compiled bundles/all_courses.scss new_styles_normal_contrast  in 381 ms
compiled bundles/all_courses.scss new_styles_high_contrast  in 381 ms
compiled bundles/attendance.scss legacy_normal_contrast  in 373 ms
compiled bundles/attendance.scss new_styles_normal_contrast  in 373 ms
compiled bundles/attendance.scss new_styles_high_contrast  in 373 ms
compiled bundles/attendance.scss legacy_high_contrast  in 374 ms
error compiling sass { [Error: Import directives may not be used within control directives or mixins.]
  cause:
   { [Error: Import directives may not be used within control directives or mixins.]
     formatted: 'Error: Import directives may not be used within control directives or mixins.\n        on line 32 of app/stylesheets/base/_variables.scss\n>>   @import "brand_variables";\n   --^\n',
     message: 'Import directives may not be used within control directives or mixins.',
     column: 3,
     line: 32,
     file: 'app/stylesheets/base/_variables.scss',
     status: 1 },
  isOperational: true,
  formatted: 'Error: Import directives may not be used within control directives or mixins.\n        on line 32 of app/stylesheets/base/_variables.scss\n>>   @import "brand_variables";\n   --^\n',
  column: 3,
  line: 32,
  file: 'app/stylesheets/base/_variables.scss',
  status: 1 } Error: Import directives may not be used within control directives or mixins.
    at options.error (/web/canvas/node_modules/brandable_css/node_modules/node-sass/lib/index.js:272:32) Import directives may not be used within control directives or mixins.
rake aborted!
Error running BrandableCSS CLI: ./node_modules/.bin/brandable_css
/web/canvas/lib/brandable_css.rb:182:in `block in run_cli!'
/web/canvas/lib/brandable_css.rb:168:in `run_cli!'
/web/canvas/lib/brandable_css.rb:148:in `compile_all!'
/web/canvas/lib/tasks/canvas.rake:121:in `block (3 levels) in <top (required)>'
/web/canvas/lib/tasks/canvas.rake:7:in `log_time'
/web/canvas/lib/tasks/canvas.rake:121:in `block (2 levels) in <top (required)>'
Tasks: TOP => canvas:compile_assets
(See full trace by running task with --trace)
```

As I understand it, according to this:

https://github.com/sass/sass/issues/451

maybe it is not such a good idea to import inside a conditional. The developer says it should have not worked in the first place and may have unintended consequences, but I am not so knowledgeable of front-end stuff.

### Steps to reproduce:

 1. bundle exec rake canvas:compile_assets in a clone of the repo

### Expected behavior:

Bundle installed.

### Additional notes:

**Comment by djbender** [comment]
it's possible this was resolved for now with 9dfe5bf? does this problem persist for you on master?

**Comment by simonista** [comment]
This should have worked previously, although we've seen an occasional issue where npm installs the wrong version of node-sass (which then manifests as this problem).  We've now fixed the underlying issue of imports inside conditionals here: ad7a8ba0700c3019835b06b602eb0d8c0adcfe86
